lab-dustin/lib/arithmetic.js

```diff
@@ -0,0 +1,24 @@
+function addFunction(num1,num2) {
+  if (typeof num1 !== 'number'
+    || typeof num2 !== 'number'
+    || isNaN(num1)
+    || isNaN(num2)
+  ){
+    return null;
+  }
+  return num1 + num2;
+}
+
+function subFunction(num1,num2) {
+  if (typeof num1 !== 'number'
+    || typeof num2 !== 'number'
+    || isNaN(num1)
+    || isNaN(num2)
+  ){
```

Comment:
As mentioned in class, this logic could be pulled out into a function to avoid repetition between add/sub.